presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -317,6 +318,11 @@ public IcebergSessionProperties(
                                 "value of 1 means a single record is equivalent to 1 millisecond of " +
                                 "time difference.",
                         icebergConfig.getStatisticSnapshotRecordDifferenceWeight(),
+                        false),
+                booleanProperty(
+                        DELETE_AS_JOIN_PUSHDOWN_ENABLED,
+                        "When enabled equality delete row filtering will be pushed down into a join.",
+                        icebergConfig.isPushdownFilterEnabled(),
```

Comment:
I believe this config value only works with the native workers at present, and that's still under development.  Let's add a new config property.

Reply:
Why would it require the native workers? I've been testing this with the Java workers I believe.

Reply:
Oh I misunderstood you, I see you were talking about the default I passed here. That was a mistake, thanks for catching that!